src/test/kotlin/no/nav/syfo/melding/cronjob/UbesvartMeldingCronjobSpek.kt

```diff
@@ -297,6 +297,32 @@ class UbesvartMeldingCronjobSpek : Spek({
                     verify(exactly = 0) { kafkaProducer.send(any()) }
                 }
 
+                it("Will not update ubesvart_published_at when melding is of type henvendelse melding fra NAV") {
```

Comment:
Er det bedre om testnavnet sier at man ikke skal lage ubesvart-oppgave av denne meldingstypen eller noe, i stedet for helt spesifikt hvilken databasekolonne som ikke endres? 🤔

Reply:
Godt poeng det - fikser!